app/jobs/cron_job.rb

```diff
@@ -0,0 +1,39 @@
+class CronJob < ApplicationJob
+  class_attribute :cron_expression
+  class_attribute :perform_on_schedule
+
+  class << self
+    def schedule
```

Comment:
This method schedule two jobs if `perform_on_schedule` is set to true: one with a `cron` value of `nil`; one with a `cron` value set to the expression defined on the class. Is that intended? I found this confusing. From my reading of the code, the impact of this is that a job would get scheduled for immediate execution every time we deployed the code, which makes me uncomfortable, especially given how long the school import takes to run. I don't believe we want or need to update the schools data on every deploy.

Could we use the [simple example base class](https://github.com/codez/delayed_cron_job#scheduling) given by the gem documentation instead? I'm not convinced we'd ever want to auto-run jobs on every deploy, and I'd feel more comfortable with the maintenance burden and potential for bugs if we weren't diverging too far from the gem's recommended base class.